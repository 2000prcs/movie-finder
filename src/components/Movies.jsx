import React, { Component } from 'react';

export default class Movies extends React.Component {
 constructor(props){
  super(props);
  this.state = {};
 }

  render() {
    return (
      <div>
        Movies
      </div>
    );
  }
}