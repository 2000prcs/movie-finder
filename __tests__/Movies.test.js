import React from 'react';
import sinon from 'sinon';
import "isomorphic-fetch";
import Movies from '../src/components/Movies.jsx';


describe('Movies component', () => {
  test('Should render Movies', () => {
    const AppWrapper = shallow(<Movies />);
    expect(AppWrapper).toMatchSnapshot();
  });

});