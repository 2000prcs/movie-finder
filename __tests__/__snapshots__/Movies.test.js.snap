// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies component Should render Movies 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Movies />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "as": "table",
      "children": Array [
        <TableHeader
          as="thead"
        >
          <TableRow
            as="tr"
            cellAs="td"
          >
            <TableHeaderCell
              as="th"
              onClick={[Function]}
              sorted={null}
              width={4}
            >
              Title
            </TableHeaderCell>
            <TableHeaderCell
              as="th"
              onClick={[Function]}
              sorted={null}
              width={3}
            >
              TMDB Rating
            </TableHeaderCell>
            <TableHeaderCell
              as="th"
              onClick={[Function]}
              sorted={null}
              width={2}
            >
              Release Date
            </TableHeaderCell>
          </TableRow>
        </TableHeader>,
        <TableBody
          as="tbody"
        />,
      ],
      "fixed": true,
      "selectable": true,
      "singleLine": true,
      "sortable": true,
      "striped": true,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "as": "thead",
          "children": <TableRow
            as="tr"
            cellAs="td"
          >
            <TableHeaderCell
              as="th"
              onClick={[Function]}
              sorted={null}
              width={4}
            >
              Title
            </TableHeaderCell>
            <TableHeaderCell
              as="th"
              onClick={[Function]}
              sorted={null}
              width={3}
            >
              TMDB Rating
            </TableHeaderCell>
            <TableHeaderCell
              as="th"
              onClick={[Function]}
              sorted={null}
              width={2}
            >
              Release Date
            </TableHeaderCell>
          </TableRow>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "as": "tr",
            "cellAs": "td",
            "children": Array [
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={4}
              >
                Title
              </TableHeaderCell>,
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={3}
              >
                TMDB Rating
              </TableHeaderCell>,
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={2}
              >
                Release Date
              </TableHeaderCell>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "as": "th",
                "children": "Title",
                "onClick": [Function],
                "sorted": null,
                "width": 4,
              },
              "ref": null,
              "rendered": "Title",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "as": "th",
                "children": "TMDB Rating",
                "onClick": [Function],
                "sorted": null,
                "width": 3,
              },
              "ref": null,
              "rendered": "TMDB Rating",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "as": "th",
                "children": "Release Date",
                "onClick": [Function],
                "sorted": null,
                "width": 2,
              },
              "ref": null,
              "rendered": "Release Date",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "as": "tbody",
          "children": Array [],
        },
        "ref": null,
        "rendered": Array [],
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "as": "table",
        "children": Array [
          <TableHeader
            as="thead"
          >
            <TableRow
              as="tr"
              cellAs="td"
            >
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={4}
              >
                Title
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={3}
              >
                TMDB Rating
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={2}
              >
                Release Date
              </TableHeaderCell>
            </TableRow>
          </TableHeader>,
          <TableBody
            as="tbody"
          />,
        ],
        "fixed": true,
        "selectable": true,
        "singleLine": true,
        "sortable": true,
        "striped": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "as": "thead",
            "children": <TableRow
              as="tr"
              cellAs="td"
            >
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={4}
              >
                Title
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={3}
              >
                TMDB Rating
              </TableHeaderCell>
              <TableHeaderCell
                as="th"
                onClick={[Function]}
                sorted={null}
                width={2}
              >
                Release Date
              </TableHeaderCell>
            </TableRow>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "as": "tr",
              "cellAs": "td",
              "children": Array [
                <TableHeaderCell
                  as="th"
                  onClick={[Function]}
                  sorted={null}
                  width={4}
                >
                  Title
                </TableHeaderCell>,
                <TableHeaderCell
                  as="th"
                  onClick={[Function]}
                  sorted={null}
                  width={3}
                >
                  TMDB Rating
                </TableHeaderCell>,
                <TableHeaderCell
                  as="th"
                  onClick={[Function]}
                  sorted={null}
                  width={2}
                >
                  Release Date
                </TableHeaderCell>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "as": "th",
                  "children": "Title",
                  "onClick": [Function],
                  "sorted": null,
                  "width": 4,
                },
                "ref": null,
                "rendered": "Title",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "as": "th",
                  "children": "TMDB Rating",
                  "onClick": [Function],
                  "sorted": null,
                  "width": 3,
                },
                "ref": null,
                "rendered": "TMDB Rating",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "as": "th",
                  "children": "Release Date",
                  "onClick": [Function],
                  "sorted": null,
                  "width": 2,
                },
                "ref": null,
                "rendered": "Release Date",
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "as": "tbody",
            "children": Array [],
          },
          "ref": null,
          "rendered": Array [],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
